lib/portage/dbapi/bintree.py

```diff
@@ -1234,7 +1234,16 @@ def _populate_local(self, reindex=True):
 
         return pkgindex if update_pkgindex else None
 
+    def _run_trust_helper(self):
+        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER")
```

Comment:
Maybe we should set a default value here too.

If we want it to succeed if the user nukes it from `make.globals` (I think I want to do this, because users might just delete it because they want something to work NOW!!!):
```suggestion
        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER", "true")
```

If we want it to default to getuto even in the source (not so sure about this - would also need some Prefix magic, but not sure if there's much point in this at all):
```suggestion
        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER", "/usr/bin/getuto")
```

Reply:
Let's use `true`. Makes for one less point of failure (and this is only a "second line of defense" anyway).